src/components/river/River.js

```diff
@@ -16,6 +18,7 @@ class River extends Component {
       level: 0,
       mapBool: false,
       symbolBool: false,
+      knowledgeBool: true,
```

Comment:
I think this should be false... the panel should not be automatically open

Reply:
Agree. I had it "true" while developing and forgot to set "false" before pushing.